packages/f-checkout/test-utils/component-objects/f-checkout.component.js

```diff
@@ -67,31 +83,41 @@ exports.populateCheckoutForm = addressInfo => {
     fields.addressLine2.input().setValue(addressInfo.line2);
     fields.addressCity.input().setValue(addressInfo.city);
     fields.addressPostcode.input().setValue(addressInfo.postcode);
+    fields.userNote.input().setValue(addressInfo.note);
 };
 
 /**
  * @description
- * Sets the value of the fulfillment time dropdown based on visible text.
- *
- * @param {String} fulfillmentTimeText The text visible text value of the fulfillment time
+ * Sets the value of the order time in dropdown based on visible text.
+ * 
+ * @param {String} orderTime The visible text value of the order time
  */
-exports.selectFulfillmentTime = fulfillmentTimeText => {
-    fulfillmentTimeDropdown().selectByVisibleText(fulfillmentTimeText);
+exports.selectOrderTime = orderTime => {
+    orderTimeDropdown().selectByVisibleText(orderTime);
+};
+/** 
+ * @description
+ * The time of the order should increase when a higher index is applied.
+ * 
+ * @param {Number} index The index of the `orderTimeDropdownOptions` array
+ */
+exports.getOrderTimeOptionText = (index) => {
+    return orderTimeDropdownOptions()[index].getText();
 };
-
 /**
  * @description
- * Sets the value of the user note.
- *
- * @param {String} userNote The user note value to be entered
+ * Sets the value of the user note and grabs the length of characters.
+ * 
+ * @param {Object} addressInfo
+ * @param {String} addressInfo.note The user's extra note
  */
-exports.inputUserNote = userNote => {
-    userNoteInput().setValue(userNote);
+exports.getUserNoteLength = addressInfo => {
```

Comment:
I think we should only make our functions do one thing, in this case returning the user note length.

If we need to fill our the user note, it should be done in another function that's called before this one in the spec.


e.g

```
// f-checkout.spec.js

.....

CheckoutComponent.inputUserNote('foo');

var noteLength = CheckoutComponent.getUserNoteLength(); // returns: 3

expect(noteLength).toBe(3))
```

We'll also need to alter the JSDoc as a result 👍